types/create_env_run_input.go

```diff
@@ -1,5 +1,6 @@
 package types
 
 type CreateEnvRunInput struct {
-	IsDestroy bool `json:"isDestroy"`
+	IsDestroy bool     `json:"isDestroy"`
+	AppIds    *[]int64 `json:"appIds"`
```

Comment:
It's odd to use a pointer to a slice.
Can we drop the pointer?

Reply:
Would it be obvious to you that an empty list of appIds means "launch them all"? I was trying to indicate that its kind of an optional parameter.

Reply:
A slice is nilable. If not specified in json or set to `null`, the slice is `nil`.

Reply:
Updated to remove the pointer.